feat: Release notes for Codacy Cloud February 2024 DOCS-631 (#2051)
codacy/docs: docs/release-notes/cloud/cloud-2024-02.md

description: Release notes for Codacy Cloud February 2024.
included_jira_versions: ['2024.02']
codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.193
codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.234
---

# Cloud February 2024

These release notes are for the Codacy Cloud updates during February 2024.

📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!

<!--TODO Check these issues manually

Jira issues without release notes

Bugs and other issues:
-   https://codacy.atlassian.net/browse/PLUTO-900

-->

## Product enhancements

-   The Codacy GitHub App no longer requires **Read and write access to Administration** repository permission. For more information, see the [Discontinuation of SSH keys for GitHub repositories](./cloud-2024-01-15-gh-repository-ssh-keys-discontinuation.md). (PLUTO-803)
-   For GitHub, if you have email privacy enabled on the provider, Codacy doesn't show your private email addresses when displaying your personal details on lists, search filters, or activity. (PLUTO-794)

> -   Added support for the Checkstyle filter [SuppressionCommentFilter](https://checkstyle.sourceforge.io/filters/suppressioncommentfilter.html) to suppress audit events. (TCE-709)

Author: Thanks, @codacy-vrhpires! Updated! 👍

> -   The [Organization overview](../../organizations/organization-overview.md) is now available for the open source plan. (ALA-842)

Reviewer: Should we mention "on all plans" as an inclusive version? Or could it confuse some users?

```suggestion
-   The [Organization overview](../../organizations/organization-overview.md) is now available on all plans. (ALA-842)
```

> -   Improved the mechanism to refresh organizations from the Git provider. As result, we removed the option to manually refresh the list of organizations when [adding new organizations to Codacy](../../organizations/what-are-organizations.md#adding-an-organization). (ALA-862)

Reviewer: Nitpick:

```suggestion
-   Improved the mechanism to refresh organizations from the Git provider. As a result, we removed the option to manually refresh the list of organizations when [adding new organizations to Codacy](../../organizations/what-are-organizations.md#adding-an-organization). (ALA-862)
```
